sopel/bot.py

```diff
@@ -376,6 +376,16 @@ def reply(self, text, dest, reply_to, notice=False):
         else:
             self.say(text, dest)
 
+    def kick(self, nick, channel, text=''):
```

Comment:
This (and below) should default the text to `None` instead of `''` to clean up the raw line generated by `irc.Bot.write()` later down the call stack.
```suggestion
    def kick(self, nick, channel, text=None):
```